### DISCUSSION TITLE
3D Labels of textlayer are getting hidden inside 3D building

### DISCUSSION THREAD

**Discussion by gireeshbhogireddy** [visgl/deck.gl#7435] [Ideas]
I want to show floor level as labels of 3D building. I am able to add labels but they are hidden if I move map in specific angle. Attaching the screenshot.

<img width="786" alt="pic" src="https://user-images.githubusercontent.com/54057993/202867623-f6f28f49-3dbe-482d-9ea8-1360edc12df2.png">

**Comment by Pessimistress** [comment]
You can disable depth testing by adding the following to the TextLayer:

```js
  parameters: {
    depthTest: false
  }
```

**Reply by gireeshbhogireddy** [reply]
Thanks @Pessimistress for the quick response, issue got resolved.
